padertorch/contrib/examples/pit/evaluate.py

```diff
@@ -154,15 +119,15 @@ def init(_config, _run):
 
 
 @ex.main
-def main(_run, batch_size, datasets, debug, experiment_dir):
+def main(_run, batch_size, datasets, debug, experiment_dir, db_path):
```

Comment:
We might want to use the same name for `db_path` in this example and the DPRNN example. In the DPRNN it is called `database_json`, and I think this better describes the value to put here than `db_path`.